docs/examples/modules/create-rg-lock-role-assignment/main.bicep

```diff
@@ -1,4 +1,4 @@
-targetScope = 'subscription'
+targetScope='subscription'
```

Comment:
This is not for this PR, but the formatter is missing a custom rule for `TargetScopeSyntax` (by default everything is concatenated together). I'll send a PR to fix it.

Reply:
Great spot, sounds good.